domains/alisabry.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "ali-sabry",
+        "email": "alisabry.official@gmail.com"
+      },
+
+      "record": {
+        "URL": "https://alisabry.is-a.dev"
```

Comment:
You can not have your is-a.dev subdomain here as it will create a redirect loop. Please add valid records.

Reply:
How can I add a valid records

Reply:
Read the [docs](https://is-a.dev/docs)